README.md

```diff
@@ -91,14 +91,16 @@ end
 ```
 
 - [ ] `Error => e`
-- [ ] *nothing*
+- [x] *nothing*
 - [ ] `SystemError => e`
-- [ ] `ArgumentError => e`
-- [ ] `StandardError => e`
-- [ ] `ArgumentError`
+- [x] `ArgumentError => e`
+- [x] `StandardError => e`
+- [x] `ArgumentError`
 - [ ] ` => ArgumentError`
 - [ ] ` => argument_error`
 
+What does $1 do here? I thought it was a regex global.
```

Comment:
Whoops! This should of been `$!`:

`When an exception has been raised but not yet handled (in rescue, ensure, at_exit and END blocks) the global variable $! will contain the current exception and $@ contains the current exception's backtrace.`